docs/launch/local-db.md

```diff
@@ -329,7 +343,9 @@ async function saveMessage(
 }
 ```
 
-### Use Effect Hook to Stream Conversations
+### Use useEffect hook to stream conversations
+
```

Comment:
A question for Ry below in HTML comment format.....

Reply:
addressed in the first comment asking this question...